src/fidesops/api/v1/urn_registry.py

```diff
@@ -68,6 +69,9 @@
 DATASETS = CONNECTION_BY_KEY + "/dataset"
 DATASET_BY_KEY = CONNECTION_BY_KEY + "/dataset/{fides_key}"
 
+# YAML Collection URLs
+YAML_DATASETS = YAML + DATASETS
```

Comment:
Nit: This will result in the URL `/api/v1/yml/connection/{key}/dataset` — could we append `YAML` here instead of prepend for the result `/api/v1/connection/{key}/dataset/yml/`, to reflect the hierarchy of what's being executed.